back-end/services/nodemailer.js

```diff
@@ -42,9 +43,18 @@ const sendMail = async (emailTo, name) => {
     },
   });
 
-  const mailOptions = registrationMailOptions(name);
+  let options;
 
-  await transporter.sendMail(mailOptions, (error, info) => {
+  switch (registrationOrSubscription) {
```

Comment:
Rename to emailType. What if we'll have a more option than 2?

Reply:
sorry, forgot about that